source/slang/hlsl.meta.slang

```diff
@@ -6041,6 +6051,16 @@ vector<T,N> normalize(vector<T,N> x)
     return x / length(x);
 }
 
+__generic<T : __BuiltinFloatingPointType>
+__target_intrinsic(hlsl)
+__target_intrinsic(glsl)
+__target_intrinsic(spirv, "OpExtInst resultType resultId glsl450 Normalize _0")
+[__readNone]
+T normalize(T x)
+{
+    return x;
```

Comment:
What exactly should this do? Should it always return 1?

Reply:
Oh! You are right. I guess it should be always 1.

Reply:
Well, should technically be just sign(x), ignoring the 0 case...